tools/check_repositories.sh

```diff
@@ -4,7 +4,7 @@ set -eu
 
 # Check whether any git repositories are defined.
 # Git repository definition contains `commit` and `remote` fields.
-if grep -nr "commit =\|remote =" --include=*.bzl .; then
+if grep -nr "commit =\|remote =" --include=*.bzl --exclude=./bazel-* .; then
```

Comment:
Thanks for working on this. From my quick look, I don't think we want to exclude this because this seems to pass OK upstream so we must be pulling in something into our repo which is causing this to fail. I think either we should figure out how to fix our repo or potentially make this exclusion list available via an ENV_VARIABLE that we can set somehow? WDYT?